docs/donate.md

```diff
@@ -18,5 +18,10 @@ donate and view how the money is being used.
   <img src="https://opencollective.com/webpack/donate/button@2x.png?color=blue" width=300 />
 </a>
 
+Additionally if you'd rather contribute to the running costs of the project via
+[GitHub Sponsors](https://github.com/sponsors/pulsar-edit) you can visit our GitHub profile to safely donate.
```

Comment:
```suggestion
If you'd rather contribute via [GitHub Sponsors](https://github.com/sponsors/pulsar-edit)
you can do this on our GitHub profile.
```

Just being picky as always, just thinking to avoid repetition a little to make it read a little easier.

Reply:
Thanks a ton, love the recommendation